lte/gateway/c/session_manager/SetMessageManagerHandler.cpp

```diff
@@ -152,6 +170,12 @@ void SetMessageManagerHandler::SetAmfSessionContext(
         /* it's new UE establisment request and need to create the session
          * context
          */
+        if (!validate_session_request(cfg)) {

```

Comment:
shouldn't this condition be checked the very first moment we receive a request. Even before building the request? meaning we should check if the request is 5g on `request_cpy`.  Is there any point of getting up to that point if the request is not 5g?

Reply:
The grpc method is used for 5g only but just put extra check for RTA_TYPE. I moved this check before build the session config.